### PR TITLE
More export conversion tweaks

### PR DESCRIPTION
**Pull request by benrudolph**
@NoahCarnahan best read 🐟

### CODE REVIEW

File: corehq/apps/export/utils.py [hunk 1]
```diff
@@ -356,11 +356,16 @@ def migrate_domain(domain, dryrun=False):
                     dryrun=dryrun
                 )
             except Exception, e:
+                raise e
                 print 'Failed parsing {}: {}'.format(old_export['_id'], e)
```
Comment (NoahCarnahan) [inline corehq/apps/export/utils.py]: I think `raise e` should go after this line

**Comment by NoahCarnahan** [comment]
Left one comment but otherwise looks good

File: corehq/apps/export/utils.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/export/utils.py]: shit i mistakenly committed that. good catch

**Comment by orangejenny** [comment]
kicked build, otherwise good to merge

**Comment by benrudolph** [comment]
closing in favor of: https://github.com/dimagi/commcare-hq/pull/12687

**Pull request closed without merging**